x-pack/plugins/uptime/common/config.ts

```diff
@@ -10,8 +10,7 @@ import { schema, TypeOf } from '@kbn/config-schema';
 import { sslSchema } from '@kbn/server-http-tools';
 
 const serviceConfig = schema.object({
-  username: schema.maybe(schema.string()),
-  password: schema.maybe(schema.string()),
+  enabled: schema.maybe(schema.boolean()),
```

Comment:
we removed this flag recently, are you adding it again fro some purpose?

Reply:
Good catch! This was probably left from a conflict when rebasing on top of `main`. Thank you 💖